packages/node_modules/@webex/internal-plugin-device/src/interceptors/device-url.js

```diff
@@ -31,7 +31,7 @@ export default class DeviceUrlInterceptor extends Interceptor {
     // Check if header is already set before moving forward
     if (
       !device.url ||
-      (headers && CISCO_DEVICE_URL in headers && !headers[CISCO_DEVICE_URL])
+      (headers && CISCO_DEVICE_URL in headers && !!headers[CISCO_DEVICE_URL])
```

Comment:
if you could, please add a small test case for this that validates that it works correctly. It should be a unit test.

Reply:
I'm realizing there is a test that already covers this, I will move this to approved.